core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -126,6 +128,8 @@ public interface ServerConfig {
      */
     Map<ChannelOption<?>, ?> childChannelOptions();
 
+    Consumer<? super ChannelPipeline> channelPipelineCustomizer();
```

Comment:
Could you add Javadoc?
```suggestion
    @UnstableApi
    Consumer<? super ChannelPipeline> channelPipelineCustomizer();
```